Wed Mar 15 14:53:49 CET 2006  VMiklos <vmiklos@frugalware.org>
  * makepkg: new directive for options(): scriptlet
diff -rN -u old-pacman/scripts/makepkg new-pacman/scripts/makepkg
--- old-pacman/scripts/makepkg	2006-03-15 15:14:01.000000000 +0100
+++ new-pacman/scripts/makepkg	2006-03-15 15:14:01.000000000 +0100
@@ -178,7 +178,7 @@
 		if [ "$DEP_BIN" = "1" ]; then
 			# install missing deps from binary packages (using pacman -S)
 			msg "Installing missing dependencies..."
-			if [ "$INCHROOT" != "1" ]; then
+			if [ "$INCHROOT" != "1" -o "`check_option SCRIPTLET`" ]; then
 				pacman $PACMAN_OPTS -D $deplist
 			else
 				pacman $PACMAN_OPTS --noscriptlet -D $deplist
@@ -196,7 +196,7 @@
 				FAKEROOTKEY2=$FAKEROOTKEY
 				unset FAKEROOTKEY
 			fi
-			if [ "$INCHROOT" != "1" ]; then
+			if [ "$INCHROOT" != "1" -o "`check_option SCRIPTLET`" ]; then
 				sudo pacman $PACMAN_OPTS -D $deplist
 			else
 				sudo pacman $PACMAN_OPTS --noscriptlet -D $deplist

